### PR TITLE
Week 7 monitoring model

### PR DESCRIPTION
**Pull request by yevhen-k**
Hi @truskovskiyk 

Here is a PR for Seldon and Arize model monitoring. The most interesting thing is that actual integration of model monitoring is not so hard as the creation and setting-up dashboards (plots, alerts, etc.) with graphana and arize UI

### CODE REVIEW

File: week-7-monitoring-model/README.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+docker tag model-monitoring:latest yevhenk10s/model-monitoring:latest && \
+docker push yevhenk10s/model-monitoring:latest
+```
+
+#### Local tests for container
+
+Start the container:
+
+```bash
+export WANDB_API_KEY=061dc0c4c510ff84584bd05674e492de2053d9da
```
Comment (truskovskiyk) [inline week-7-monitoring-model/README.md]: remove key please

File: week-7-monitoring-model/README.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+```bash
+python ../week-4-kuberflow-pipelines/load-data/download-dataset.py --n-samples=300 --noise=0.3 --random-state=0 --out-file=training-dataset.json
+
+python ../week-4-kuberflow-pipelines/load-data/download-dataset.py --n-samples=300 --noise=0.6 --random-state=42 --out-file=validation-dataset.json
+```
+
+2. Download pretrained model
+
+```bash
+export WANDB_API_KEY=bc80cf5faf07246e0634fcc56ca1e241075d38e6
```
Comment (truskovskiyk) [inline week-7-monitoring-model/README.md]: remove this please

File: week-7-monitoring-model/arize-client-demo.py [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+    arize_client = Client(space_key=SPACE_KEY, api_key=API_KEY)
+    # Saving model metadata for passing in later
+    model_id = "random_forest_clf"
+    model_version = "latest"
+    print("Step 2: Import and Setup Arize Client Done! Now we can start using Arize!")
+
+    log_train_dataframe(train_df, model_id, model_version, arize_client)
+    log_val_dataframe(val_df, model_id, model_version, arize_client)
+    log_test_dataframe(test_df, model_id, model_version, arize_client)
+    print("DONE")
```
Comment (truskovskiyk) [inline week-7-monitoring-model/arize-client-demo.py:180]: nice!

**Review by truskovskiyk — COMMENTED** [review]
@yevhen-k looks good, could you add screenshots of your dashboard, please?

File: week-7-monitoring-model/README.md (hunk 1 above)
Comment (yevhen-k) [reply week-7-monitoring-model/README.md]: done

File: week-7-monitoring-model/README.md (hunk 2 above)
Comment (yevhen-k) [reply week-7-monitoring-model/README.md]: done

**Review by truskovskiyk — APPROVED** [review]

**Pull request merged**